build-system-tests/scripts/mega-app-copy-files.sh

```diff
@@ -160,16 +160,8 @@ if [[ "$FRAMEWORK" == 'angular' ]]; then
 fi
 
 if [[ "$FRAMEWORK" == 'vue' ]]; then
-    echo "cp templates/components/vue/App.vue mega-apps/${MEGA_APP_NAME}/src/App.vue"
-    cp templates/components/vue/App.vue mega-apps/${MEGA_APP_NAME}/src/App.vue
-    echo "cp $AWS_EXPORTS_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.js"
-    cp $AWS_EXPORTS_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.js
-    echo "cp $AWS_EXPORTS_DECLARATION_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.d.ts"
-    cp $AWS_EXPORTS_DECLARATION_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.d.ts
-
-    # remove comments from JSON files because `json` package can't process comments
-    echo "npx strip-json-comments mega-apps/${MEGA_APP_NAME}/tsconfig.app.json >tmpfile && mv tmpfile mega-apps/${MEGA_APP_NAME}/tsconfig.app.json && rm -f tmpfile"
-    npx strip-json-comments mega-apps/${MEGA_APP_NAME}/tsconfig.app.json >tmpfile && mv tmpfile mega-apps/${MEGA_APP_NAME}/tsconfig.app.json && rm -f tmpfile
```

Comment:
Info: why is this no longer needed?

Reply:
This was supposed to fix an issue with the `json` package, which was being used to add `allowJs=true` to the package.json, and that is no longer needed. (Not sure why in that case but there's a decent amount of legacy code in here so I assume it was a problem at one point and isn't anymore)
(see https://github.com/aws-amplify/amplify-ui/pull/5746#discussion_r1744587644)

Reply:
Ah, ok. Do you know if `strip-json-comments` is still needed for Angular as well? (line 151)

Reply:
Just tested and seems like it is still needed. https://github.com/aws-amplify/amplify-ui/actions/runs/10725378615/job/29743057705